Documentation/guides/Extending LiveCode.md

```diff
@@ -614,6 +614,54 @@ support/icon.png // 20x40 png image
 support/icon@extra-high.png // 40x80 png image
 ```
 
+## The Extension Store
+
+Once you have created and packaged your extension you can make it available to the LiveCode Community via the Extension Store.
```

Comment:
Do you think it makes sense for "Extension Store" in this paragraph to be a hyperlink?